docs/pages/installation/kubernetes.md

```diff
@@ -4,51 +4,62 @@ title: Kubernetes
 permalink: installation/platforms/kubernetes
 ---
 
-# Quick Start with Kubernetes
+## Quick Start with Kubernetes
+
+To set up and run Meshery on Kubernetes:
+
+- <a href="#helm">Use Helm and set up a Kubernetes cluster</a>
+
+- <a href="#k8s">Run Meshery on existing Kubernetes cluster</a>
   
-<a name="helm"></a>
-## Using Helm
+<a name="helm">
```

Comment:
These anchors are duplicative and should be removed. Anchors are autogenerated for each heading.